sites.json

```diff
@@ -1,5 +1,15 @@
 [
     {
+        "name": "1Password",
+        "url": "http://my.1password.com/profile",
```

Comment:
I don’t have a 1Password account, but can you just confirm that’s the URL you’d use? One thing I notice going to that page, it redirects to the https scheme, should this URL be using https?

Thanks